app/client/components/GridView.ts

```diff
@@ -2387,7 +2388,10 @@ export default class GridView extends BaseView {
 
     bundleChanges(() => {
       fields.forEach((field, index) => {
-        const maxWidth = fieldMaxWidths.get(index) || 100;
+        const recordedWidth = fieldMaxWidths.get(index) || 100;
+        // If field can be wrapped, we will use 100 as minimum width.
```

Comment:
Why do we want to make this choice for our users, even when they are intentionally trying to make a column narrower? E.g. in Team Check-ins doc, we have columns less than 100 that are wrapped.